submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+    <link rel="stylesheet" href="css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="page_header">
+      <button type="button" class="button page_header_button_back">
+        <h4 class="page_header_button_back_text">Go back</h4>
+      </button>
+      <img src="images/logo-header.png" alt="logo" class="header_logo" />
+      <input
+        type="button"
+        value="Sign in"
+        class="button page_header_button_forward"
+      />
+    </header>
+    <main>
+      <form action="#" class="form">
+        <section class="form-header">
+          <section class="form-header_status">
+            <div class="wrapper_form-header_status_content">
+              <h4 class="form-header_titles form-header_status_title_text">
+                Your order
+              </h4>
+              <table class="form-header_info_table">
+                <thead>
+                  <th class="form-header_status_table_cell">ITEMS</th>
+                  <th class="form-header_status_table_cell">DETAILS</th>
+                  <th class="form-header_status_table_cell">PRICE</th>
+                  <th class="form-header_status_table_cell">DELIVERY</th>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="form-header_status_table_cell">2 documents</td>
+                    <td class="form-header_status_table_cell">2 languages</td>
+                    <td class="form-header_status_table_cell">$696.06</td>
+                    <td class="form-header_status_table_cell">48 hours*</td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </section>
+            <section class="wrapper_form-header_status_buttons">
+              <button
+                type="submit"
+                class="button submit_button form-header_submit_button"
+              >
+                Place order
+              </button>
+              <button type="button" class="button form-header_print_button">
+                Print a quote
+              </button>
+            </section>
+          <section class="form-header_info">
+            <div class="wrapper_form-header_info_content">
+              <h4 class="form-header_titles form-header_info_title">
+                Help center
+              </h4>
+              <p class="form-header_info_text">
+                Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet.Lorem
+                ipsum dolor sit amet. Lorem ipsum dolor sit amet.Lorem ipsum
+                dolor sit amet.Lorem ipsum dolor sit amet.
+              </p>
+              <div class="wrapper_form-header_info_contacts">
+                <h4>+971 (0) 555 555</h4>
+                <h5 class="form-header_info_email">
```

Comment:
Use corresponding HTML tag for the emails. [Read more](https://developer.mozilla.org/ru/docs/Web/HTML/Element/A#%D0%A1%D0%BE%D0%B7%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F_%D1%81%D1%81%D1%8B%D0%BB%D0%BA%D0%B8_%D0%B4%D0%BB%D1%8F_%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BD%D0%B8%D1%8F_%D0%BF%D0%B8%D1%81%D1%8C%D0%BC%D0%B0)

Reply:
Thank's for the link.Done